bcipy/helpers/tests/test_raw_data.py

```diff
@@ -0,0 +1,272 @@
+"""Tests for BciPy raw data format."""
+import os
+import shutil
+import tempfile
+import unittest
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from bcipy.helpers.raw_data import (RawData, RawDataReader, RawDataWriter,
+                                    load, sample_data, settings, write)
+
+
+class TestRawData(unittest.TestCase):
+    """Tests for raw_data format"""
+
+    def setUp(self):
+        """Override; set up the needed path for load functions."""
+        self.data_dir = f"{os.path.dirname(__file__)}/resources/"
+        self.temp_dir = tempfile.mkdtemp()
+
+    def tearDown(self):
+        """Override"""
+        shutil.rmtree(self.temp_dir)
+
+    def test_init(self):
+        """Test that a RawData structure can be initialized"""
+        columns = ['timestamp', 'ch1', 'ch2', 'ch3', 'TRG']
+        data = RawData(daq_type='DSI-24', sample_rate=300.0, columns=columns)
+        self.assertEqual(data.daq_type, 'DSI-24')
+        self.assertEqual(data.sample_rate, 300.0)
+        self.assertEqual(data.columns, columns)
+
+    def test_write_with_no_data(self):
+        """Test that raw data can be persisted to disk."""
+        path = Path(self.temp_dir, 'test_raw_data.csv')
+        data = RawData(daq_type='DSI-24',
```

Comment:
There are some repeated steps in many of these unittests. It may be more change resilient to bring path/data up to `self` and reconstruct them when required (using the LSL daq type, etc)